<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_main</class>
 <widget class="QWidget" name="form_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>582</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>999</width>
    <height>999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Context Free Grammar Simulator</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>621</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Log das operações</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>621</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTextEdit" name="text_log">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>371</width>
     <height>449</height>
    </rect>
   </property>
   <property name="title">
    <string>Parser descendente recursivo</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>371</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextEdit" name="text_parser">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="line_sentence"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_recognize">
         <property name="text">
          <string>Reconhecer sentença</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>241</width>
     <height>449</height>
    </rect>
   </property>
   <property name="title">
    <string>Gramática livre de contexto</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>241</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btn_read">
       <property name="text">
        <string>Ler de arquivo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_save">
       <property name="text">
        <string>Salvar em arquivo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="text_grammar"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_verify">
       <property name="text">
        <string>Verificar se é LL(1)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_parser">
       <property name="text">
        <string>Gerar parser descente recursivo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>groupBox_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
